readme.md

```diff
@@ -345,6 +345,21 @@ _arg => array.map(_arg)
 array.map(_it => _it)
 ```
 
+Likewise, only at the top-level and as the right-hand side of an assignment does `it` and `_` behave similarly when used to produce implicit n-ary functions — since there's no further upward to go they can both stop at the same place.
+
+The following two map implementations do the same thing:
+
+```js
+import { _, it } from 'param.macro'
+
+const map1 = _.map(_)
+const map2 = it.map(_)
+```
+
+_However_, if nested deeper inside a function call, the object placeholder `_` would traverse upward and create a separate function _first_, before being passed the `_` inside the method call itself. This creates a unary method call instead of the implicit binary function we wanted, lift or not.
```

Comment:
```suggestion
_However_, if nested deeper inside a function call the object placeholder `_` in
`map1` above would traverse further upward than an `it` would, and create a separate
function _first_, before the argument placeholder `_` inside the method call itself.
This creates an unary method call instead of the implicit binary function we probably
wanted, `lift` or not.
```